docs/dev/build.md

```diff
@@ -21,14 +21,20 @@ This charpter describe how to prepare your machine for OpenVINO development.
 ### Software requirements 
```

Comment:
why do we need a big documents with how to build for all OSes?
what is the purpose of this? 
since the documents are different, why not to split them into dedicated parts like it's on Wiki?

Reply:
Let's try to split it on several files and check the result

Reply:
I agree that so big file is hard to maintain

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/a0e9cdaa3c860f5c99928428493254373933a0e1